lib/src/main/java/org/openapitools/openapistylevalidator/OpenApiSpecStyleValidator.java

```diff
@@ -131,7 +131,8 @@ private void validateModelProperties(String modelName, Schema model) {
             model.getProperties().forEach((propertyName, property) -> {
                 if (parameters.isValidateModelPropertiesExample() && property.getExample() == null
                         && ((property.getItems() == null && property.getRef() == null)
-                            || (property.getItems() != null && property.getItems().getRef() == null))) {
+                            || (property.getItems() != null && property.getItems().getRef() == null)) &&
+                            property.getAllOf()==null) {
```

Comment:
Please do a little reformat here with space before and after the `==`.
Like this:
```suggestion
                            property.getAllOf() == null) {
```